apps/site/tests/e2e/general-behavior.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { importLocale } from '@node-core/website-i18n';
+import { test, expect, type Page } from '@playwright/test';
+
+// TODO(@AvivKeller): It would be ideal for all the Test IDs to not exist in the
+//                    ui-components package, and instead be passed as props.
+const selectors = {
```

Comment:
Nit: I think that `testIds` or something like that would be more appropriate at this point

Reply:
I feel like `selectors` still works since this contains both `testIds` and (for Orama) element names.

Reply:
ah ok I see, I missed that :+1: 

(but now I kind of wonder if including both test-ids and standard selectors in this object is potentially confusing :thinking:)

Reply:
I think it'll be fine, as I have a comment stating they are IDs.